waku/v2/utils/wakuswitch.nim

```diff
@@ -49,6 +87,9 @@ proc newWakuSwitch*(
     if wsEnabled == true:
       b = b.withAddresses(@[wsAddress, address])
       b = b.withWsTransport()
+    if wssEnabled == true:
```

Comment:
Should we warn if someone enables wss and ws at same time? Not sure what is most common here. Similar to "https only"

Reply:
added debug log